cmd/root.go

```diff
@@ -214,6 +243,43 @@ func initConfig() {
 	rootConfig.Debug = (viper.GetInt("verbose") >= 1)
 	rootConfig.Timeout = (viper.GetInt("timeout"))
 
+	clientCertContent := ""
+
+	if viper.GetString("tls-client-cert") != "" {
+		clientCertContent = viper.GetString("tls-client-cert")
+	} else if viper.GetString("tls-client-cert-file") != "" {
+		fileContent, err := ioutil.ReadFile(viper.GetString("tls-client-cert-file"))
+		if err != nil {
+			fmt.Printf("read file %q: %s", viper.GetString("tls-client-cert-file"), err)
+			os.Exit(1)
+		}
+		clientCertContent = string(fileContent)
+		clientCertContent = strings.TrimRight(clientCertContent, "\n")
+	}
+	rootConfig.TLSClientCert = clientCertContent
+
+	clientKeyContent := ""
+
+	if viper.GetString("tls-client-key") != "" {
+		clientKeyContent = viper.GetString("tls-client-key")
+	} else if viper.GetString("tls-client-key-file") != "" {
+		fileContent, err := ioutil.ReadFile(viper.GetString("tls-client-key-file"))
+		if err != nil {
+			fmt.Printf("read file %q: %s", viper.GetString("tls-client-key-file"), err)
```

Comment:
Get the value from viper once and use it in all places. Best done to avoid typos.

Reply:
Will do.